infinitic-storage-mysql/src/main/kotlin/io/infinitic/storage/mysql/MySQLKeyCounterStorage.kt

```diff
@@ -0,0 +1,94 @@
+/**
+ * "Commons Clause" License Condition v1.0
+ *
+ * The Software is provided to you by the Licensor under the License, as defined
+ * below, subject to the following condition.
+ *
+ * Without limiting other conditions in the License, the grant of rights under the
+ * License will not include, and the License does not grant to you, the right to
+ * Sell the Software.
+ *
+ * For purposes of the foregoing, “Sell” means practicing any or all of the rights
+ * granted to you under the License to provide to third parties, for a fee or
+ * other consideration (including without limitation fees for hosting or
+ * consulting/ support services related to the Software), a product or service
+ * whose value derives, entirely or substantially, from the functionality of the
+ * Software. Any license notice or attribution required by the License must also
+ * include this Commons Clause License Condition notice.
+ *
+ * Software: Infinitic
+ *
+ * License: MIT License (https://opensource.org/licenses/MIT)
+ *
+ * Licensor: infinitic.io
+ */
+
+package io.infinitic.storage.mysql
+
+import com.zaxxer.hikari.HikariDataSource
+import io.infinitic.common.storage.keyCounter.KeyCounterStorage
+import org.jetbrains.annotations.TestOnly
+
+private const val MYSQL_TABLE = "infinitic.key_counter_storage"
```

Comment:
Maybe we want to have the database/schema in configuration?

Reply:
Very true, but we have to make the database name a mandatory information in the config then !

Reply:
I agree that the table name should be in config

Reply:
I changed this part, database name is now used only at connection time in the JDBC url (and it takes the database name from the config)